### DISCUSSION TITLE
Orbit Line Not Drawn

### DISCUSSION THREAD

**Discussion by nicklucas** [Stellarium/stellarium#3521] [General]
![stellarium-001](https://github.com/Stellarium/stellarium/assets/12992734/50f77744-a1ea-4f56-bf0c-f76e01b544f9)

My iMac suffered a major failure recently after 8 years of no problems. I re-installed Stellarium but the ISS orbits are not being drawn. I've check the .json file in the application support and the orbit visible is set to true. I thought this would be easy to find/solve using Google but I'm at a loss. Can anyone help this poor frustrated old fool ?

**Comment by alex-w** [comment]
Please check Settings tab in Satellites plugin.

**Reply by nicklucas** [reply]
Settings are as I would expect. The check boxes for display and for orbit are both checked for Zarya. I also double checked the satellite.json to see that i updated correctly if the show orbit was not checked. It chaned to false when not checked and true when checked. I did also remove and reload the entire application.

**Reply by gzotti** [reply]
On the first tab of the satellite plugin, there is the global switch to activate drawing of all satellite orbits where orbit drawing is activated.

**Reply by nicklucas** [reply]
![stellarium-002](https://github.com/Stellarium/stellarium/assets/12992734/4a693d67-4dcb-4c2d-a1ac-b07863009ff1)
I also had to reset all existing satellites to prevent the entire screen being covered with orbit lines. On restarting Stellarium the display is back to what I was expecting. Thank you so much for your help and patience. I do hope that this helps anyone else with a similar issue.
